test/factories/compute_resources.rb

```diff
@@ -49,7 +49,7 @@
       provider 'Vmware'
       user 'vuser'
       password 'vpassword'
-      sequence(:url) { |n| "#{n}.example.com" } # alias for server
```

Comment:
As written above, you cannot use `sequence(:server)` here as FactoryGirl doesn't allow you to use an aliased method. Please use `sequence(:url)` and restore the comment.